pin/player.cpp

```diff
@@ -4914,9 +4914,15 @@ void Player::Render()
    if (m_pactiveball) // Debug Ball Pos
    {
       Vertex3Ds pos = m_pactiveball->m_d.m_pos;
-      Vertex3Ds velo = m_pactiveball->m_d.m_vel;
-      std::string posXYZ = "posX: " + float_to_str(pos.x) + ", " + "posY " + float_to_str(pos.y) + ", " + "pos Z: " + float_to_str(pos.z);
-      std::string veloXYZ = "VeloX: " + float_to_str(velo.x) + ", " + "VeloY " + float_to_str(velo.y) + ", " + "VeloZ: " + float_to_str(velo.z);
+      Vertex3Ds vel = m_pactiveball->m_d.m_vel;
+      std::string posAndVelXYZ = 
+          float_to_str(pos.x) + "," + 
+          float_to_str(pos.y) + "," + 
+          float_to_str(pos.z) + "," +
+          float_to_str(vel.x) + "," + 
+          float_to_str(vel.y) + "," + 
+          float_to_str(vel.z);
+      /*
       DebugPrint(0, 80, "Pos: ", false);
```

Comment:
Instead of commenting out, maybe set up a logger? That way you can change logging level in one place and the debug prints will show or not show.

Reply:
Fixed that